File: py/ege25.78.py
from math import sqrt
n = 0
M = 20000
for i in range(2, M + 1):
    prime = 1
    for x in range(2, int((sqrt(i)) + 1)):
        if i % x == 0:
            prime = 0
            break
    if prime:
        n += 1

print(n)

File: c/ege25.78.c
#include <stdio.h>

int main() {
    int N = 3577000;
    int n = 0;

    for(int i = 2; i <= N; i++) {
        int prime = 1;
        for(int j = 2; j * j <= i; j++) {
            if(i % j == 0) {
                prime = 0;
                break;
            }
        }

        if(prime)
            n++;
    } 

    printf("%d\n", n);
}

